.travis.yml

```diff
@@ -1,18 +1,55 @@
+sudo: required
```

Comment:
If I remember correctly, It was needed to upgrade docker to a custom version. Could be, with a little work to the Docker Compose pieces below, that this could be removed. I could be wrong though.